js/jquery/plugins/jquery.scrollTo.js

```diff
@@ -1,7 +1,7 @@
 /**
- * Copyright (c) 2007-2014 Ariel Flesler - aflesler<a>gmail<d>com | http://flesler.blogspot.com
+ * Copyright (c) 2007 Ariel Flesler - aflesler ○ gmail • com | https://github.com/flesler
```

Comment:
I'm worried about the rendering of the characters... are you using Windows?

Reply:
Yes, I am on Windows. I will look in to that.

Reply:
Solved, nice catch.